styles/styles.css

```diff
@@ -677,28 +684,20 @@ main .section.small-vertical-margins {
     padding-top: 32px;
 }
 
-.copd-resources main .default-content-wrapper {
-    line-height: 1.2em;
-    font-size: 1.4rem;
-}
-
-.copd-resources main .default-content-wrapper p {
-    line-height: 1.2em;
-}
-
-.copd-resources main .default-content-wrapper h2 {
+.copd-resources main div:not(.isi) .default-content-wrapper h2,
+.getting-started main div:not(.isi) .default-content-wrapper h2 {
     font-size: 2rem;
```

Comment:
font size on mobile should be 23px, also I think we should use the variable

Reply:
margin-top: 25px and margion-bottom: 26px, the spacing between the h2 and the paragraph should be a bit bigger, on mobile

Reply:
I am aware of these differences, see below.

h2 size is different on different pages, hence I choose to keep 2rem
I agree we should use vars but at the moment we don't have the right vars and all their variants correctly; hence proceeding with fixed values. opened a refactoring issue for that

I am addressing margins as a global review in https://github.com/hlxsites/bevespi/issues/114